octproz/src/glwindow2d.h

```diff
@@ -131,7 +131,15 @@ class GLWindow2D : public QOpenGLWidget, protected QOpenGLFunctions
 	//OctAlgorithmParameters::DISPLAY_FUNCTION displayFuntion;
 	unsigned int markerPosition;
 	bool markerVisible;
-
+    //rotation
+    float screenWidthScaled01;
+    float screenWidthScaled11;
+    float screenWidthScaled10;
+    float screenWidthScaled00;
+    float screenHeightScaled01;
+    float screenHeightScaled11;
+    float screenHeightScaled10;
+    float screenHeightScaled00;

```

Comment:
humble request: could you use tabs instead of spaces, please? Just to keep the code more or less consistent.